### PR TITLE
[CN] Update translation of Lesson 01

### PR DESCRIPTION
**Pull request by kj415j45**

### CODE REVIEW

**Comment by kj415j45** [comment]
@microsoft-github-policy-service agree

File: 01-introduction-to-genai/translations/cn/README.md [hunk 1]
```diff
@@ -117,9 +117,9 @@
 
 ## 任务
 
-你的任务是阅读更多关于生成式人工智能（https://en.wikipedia.org/wiki/Generative_artificial_intelligence）的内容，并尝试找出一个你现在可以添加生成式人工智能但目前还没有的领域。 与用“旧式方法”做有什么不同，你能做一些你以前做不到的事情吗，或者生成的时候速度更快吗？ 写一篇 300 字的摘要，描述您梦想的人工智能初创公司的样子，包括“问题”、“我将如何使用人工智能”、“影响”等标题，还可以选择做一份商业计划。
+你的任务是阅读更多关于[生成式人工智能](https://en.wikipedia.org/wiki/Generative_artificial_intelligence)的内容，并尝试找出一个你现在可以添加生成式人工智能但目前还没有的领域。 与用“旧式方法”做有什么不同，你能做一些你以前做不到的事情吗，或者生成的时候速度更快吗？ 写一篇 300 字的摘要，描述您梦想的人工智能初创公司的样子，包括“问题”、“我将如何使用人工智能”、“影响”等标题，还可以选择做一份商业计划。
 
-如果您完成了此任务，您甚至可以准备好申请 Microsoft 的创业孵化器[Microsoft for Startups Founders Hub](https://www.microsoft.com/startups?WT.mc_id=academic-105485-koreyst)，我们为您提供 Azure 和 OpenAIc redits 以及相关指导等等
+如果您完成了此任务，您甚至可以准备好申请 [Microsoft 的创业孵化器\(Microsoft for Startups Founders Hub\)](https://www.microsoft.com/zh-cn/startups?WT.mc_id=academic-105485-koreyst)，我们为您提供 Azure 和 OpenAI 的免费额度以及相关指导等等。
```
Comment (kj415j45) [inline 01-introduction-to-genai/translations/cn/README.md:122]:
```suggestion
如果您完成了此任务，您甚至可以准备好申请 [Microsoft 的创业孵化器\(Microsoft for Startups Founders Hub\)](https://www.microsoft.com/startups?WT.mc_id=academic-105485-koreyst)，我们为您提供 Azure 和 OpenAI 的免费额度以及相关指导等等。
```

File: 01-introduction-to-genai/translations/cn/README.md [hunk 2]
```diff
@@ -34,13 +34,13 @@
...
 
-![Five young students looking at a monitor - image by DALLE2](../../images/students-by-DALLE2.png?WT.mc_id=academic-105485-koreyst)
+![五位学生正看着屏幕 - 由 DALLE2 生成的图片](../../images/students-by-DALLE2.png?WT.mc_id=academic-105485-koreyst)
 
 首先，让我们来学习一些将在整个课程系列中使用的基本概念和术语。
 
 ## 我们是如何获得生成式人工智能的？
 
-尽管最近因生成人工智能模型的发布而引起了对人工智能技术的“炒作”，但人工智能已经有数十年的历史，最早的研究工作可以追溯到上世纪 60 年代。 我们现在正处于 AI 具有人类认知能力的阶段，例如 [OpenAI ChatGPT](https://openai.com/chatgpt?WT.mc_id=academic-105485-koreyst) 或 [Bing Chat](https://www.microsoft.com/en-us/edge/features/bing-chat?WT.mc_id=academic-105485-koreyst）也在用 GPT 模型进行对话。
+尽管最近因生成人工智能模型的发布而引起了对人工智能技术的“炒作”，但人工智能已经有数十年的历史，最早的研究工作可以追溯到上世纪 60 年代。我们现在正处于 AI 具有人类认知能力的阶段，例如 [OpenAI ChatGPT](https://openai.com/chatgpt?WT.mc_id=academic-105485-koreyst) 或 [Bing Chat](https://www.microsoft.com/en-us/edge/features/bing-chat?WT.mc_id=academic-105485-koreyst) 也在用 GPT 模型进行对话。
```
Comment (kj415j45) [inline 01-introduction-to-genai/translations/cn/README.md:43]:
```suggestion
尽管最近因生成人工智能模型的发布而引起了对人工智能技术的“炒作”，但人工智能已经有数十年的历史，最早的研究工作可以追溯到上世纪 60 年代。我们现在正处于 AI 具有人类认知能力的阶段，例如 [OpenAI ChatGPT](https://openai.com/chatgpt?WT.mc_id=academic-105485-koreyst) 或 [Bing Chat](https://www.microsoft.com/edge/features/bing-chat?WT.mc_id=academic-105485-koreyst) 也在用 GPT 模型进行对话。
```

**Comment by PAN740623** [comment]
阻止报告

**Review by skydiver1980 — APPROVED** [review]

**Pull request closed without merging**